cat-home/src/test/java/com/dianping/cat/report/task/TaskHelperTest.java

```diff
@@ -53,6 +58,12 @@ public void testYesterdayZero() {
 		Assert.assertEquals(cal.getTimeInMillis(), yesterdayZero.getTime() + 3600L * 1000 * 24);
 	}
 
+    @Test
+    public void testYesterdayZero2() {
```

Comment:
Mocking is needed for this test as the day changes

Reply:
Pass a long value when creating a `new Date()`

Reply:
mocking is done